search-relevancy-test/src/search_relevancy_test/reporter.clj

```diff
@@ -39,8 +73,52 @@
           (when (= -1 result-position)
             (println (ansi/style (format "...............Concept %s was not found in the result set"
                                          concept-id)
-                                 :magenta)))))
-      (if (= (sort items-in-positional-order) items-in-positional-order)
-        (println (ansi/style (format "%s OK" description) :green))
-        (println (ansi/style (str description " FAILED - out of order") :red)
-                 colored-items)))))
+                                 :magenta))))
```

Comment:
We should get rid of `atom-fail-count (atom 0)` in the let above. I had that in a branch that I guess I never merged.